packages/jest-changed-files/src/hg.ts

```diff
@@ -20,7 +20,7 @@ const adapter: SCMAdapter = {
 
     const args = ['status', '-amnu'];
     if (options && options.withAncestor) {
-      args.push('--rev', `min((!public() & ::.)+.)^`);
+      args.push('--rev', `min((!public() & ::.)+.)`);
```

Comment:
interesting! I've never used mercurial before, but since the test assert on this I guess the change is correct? 🙂

Reply:
My first time with Mercurial, to be honest. That caret symbol might be needed, but seems like this is the trouble maker. Possibly Execa is somehow escaping it.